pkg/storage/etcd/testing/utils.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package etcd
+package testing
```

Comment:
I kind of assumed that renaming a package implied renaming the directory... It's kind of silly to have a package that is only used in unit tests that name collides with the one package that is imported in _all_ unit tests.